src/dict.c

```diff
@@ -1817,7 +1819,9 @@ int dictTest(int argc, char **argv, int flags) {
     }
 
     TEST("Delete one more key, trigger the dict resize") {
-        retval = dictDelete(dict,stringFromLongLong(68));
+        char *key = stringFromLongLong(68);
+        retval = dictDelete(dict, key);
+        zfree(key);
```

Comment:
How about this, for the constant values?

```suggestion
        retval = dictDelete(dict, "68");
```

Ah, already merged. Doesn't matter. :-)